singleton-service/Cargo.toml

```diff
@@ -9,3 +9,9 @@ crate-type = ["cdylib"]
 
 [dependencies]
 threescale = { path = "../threescale" }
+serde = { version = "1.0.126", features = ["derive"] }
+serde_json = "1.0.64"
+log = "0.4.13"
+serde-humanize-rs = "0.1.1"
+proxy-wasm = "0.1.3"
```

Comment:
We maintain a fork with some fixes (and a few incompatibilities) which might be interesting to you [here](https://github.com/3scale/proxy-wasm-rust-sdk).

Reply:
okay i'll have a look.